kubemarine/plugins/calico.py

```diff
@@ -105,6 +105,12 @@ def apply_calico_yaml(cluster, calico_original_yaml, calico_yaml):
     cluster.log.verbose(f"The total number of excluded objects is {len(excluded_list)} "
                         f"the objects are the following: {excluded_list}")
 
+    # put this file into a dump_location folder or in /tmp if dump is disabled
+    if not cluster.context['execution_arguments']['disable_dump']:
+        calico_yaml = cluster.context['execution_arguments']['dump_location'] + "/" + calico_yaml
+    else:
+        calico_yaml = '/tmp/' + calico_yaml
```

Comment:
This will not work when deploying from windows. Also ideally, it is not necessary to save data as a file and could be just transferred to remote nodes directly from memory. I have a chance to write smth like plugins.apply_stream but not finished https://github.com/Netcracker/KubeMarine/commit/5c4566888da2e7dddec138e1f9416a25c274aad7